interface/src/app/gene/searchMany/searchMany.js

```diff
@@ -25,6 +25,15 @@ angular.module('adage.gene.searchMany', [
               // be obtained from the ML model. This is the same as
               // the issue in geneSearchForm (also with $scope.organism).
               $scope.organism = 'Pseudomonas aeruginosa';
+
+              // '$scope.autocomplete.autocomlete' holds a boolean value,
+              // of whether or not the user wants to use autocomplete
+              // search to look up a few genes. If not, then the autocomplete
+              // search panel will disappear, and the panel to search many
+              // genes will appear. We need to send the value itself inside
+              // an object so that it gets sent properly back and forth
+              // through the corresponding $scopes.
+              $scope.autocomplete = {autocomplete: true};
```

Comment:
Thanks for that @dongbohu - I tried out the example you sent, and I noticed that it uses controllerAs syntax, so I switched the code in here to also use controllerAs syntax instead of $scope - this worked! I am still scratching my head though, as to why this works, just like putting the boolean inside a new object, as opposed to just binding a boolean flag to the $scope object. Following the Pragmatic Programmer's mantra, I feel like this is a bug until I fully understand it, so I am diving into the documentation about this. Do you have any insights on this?